test/std-format-test.cc

```diff
@@ -102,7 +102,7 @@ template <> struct std::formatter<S> {
   size_t width_arg_id = 0;
 
   // Parses a width argument id in the format { <digit> }.
-  constexpr auto parse(format_parse_context& ctx) {
+  auto parse(format_parse_context& ctx) {
```

Comment:
What is this for?

Also could you move the fixes to a separate PR and post the warnings they are fixing for future reference?

Reply:
Detailed writeup in the commit notes here: https://github.com/fmtlib/fmt/pull/1999/commits/34ea1472eb8dd2463819bdcfb4b81b6f64374cc3

I'll split these all out to individual PRs.

Reply:
Created #2008, #2009, and #2010 to resolve the warnings.